src/Compilers/CSharp/Test/Symbol/Symbols/UnsignedRightShiftTests.cs

```diff
@@ -0,0 +1,1873 @@
+        [InlineData("ulong", "decimal")]
+        [InlineData("nint", "object")]
+        [InlineData("nint", "string")]
+        [InlineData("nint", "bool")]
+        [InlineData("nint", "long")]
+        [InlineData("nint", "uint")]
+        [InlineData("nint", "ulong")]
+        [InlineData("nint", "nint")]
+        [InlineData("nint", "nuint")]
+        [InlineData("nint", "float")]
+        [InlineData("nint", "double")]
+        [InlineData("nint", "decimal")]
+        [InlineData("nuint", "object")]
+        [InlineData("nuint", "string")]
+        [InlineData("nuint", "bool")]
+        [InlineData("nuint", "long")]
+        [InlineData("nuint", "uint")]
+        [InlineData("nuint", "ulong")]
+        [InlineData("nuint", "nint")]
+        [InlineData("nuint", "nuint")]
+        [InlineData("nuint", "float")]
+        [InlineData("nuint", "double")]
+        [InlineData("nuint", "decimal")]
+        [InlineData("float", "object")]
+        [InlineData("float", "string")]
+        [InlineData("float", "bool")]
+        [InlineData("float", "char")]
+        [InlineData("float", "sbyte")]
+        [InlineData("float", "short")]
+        [InlineData("float", "int")]
+        [InlineData("float", "long")]
+        [InlineData("float", "byte")]
+        [InlineData("float", "ushort")]
+        [InlineData("float", "uint")]
+        [InlineData("float", "ulong")]
+        [InlineData("float", "nint")]
+        [InlineData("float", "nuint")]
+        [InlineData("float", "float")]
+        [InlineData("float", "double")]
+        [InlineData("float", "decimal")]
+        [InlineData("double", "object")]
+        [InlineData("double", "string")]
+        [InlineData("double", "bool")]
+        [InlineData("double", "char")]
+        [InlineData("double", "sbyte")]
+        [InlineData("double", "short")]
+        [InlineData("double", "int")]
+        [InlineData("double", "long")]
+        [InlineData("double", "byte")]
+        [InlineData("double", "ushort")]
+        [InlineData("double", "uint")]
+        [InlineData("double", "ulong")]
+        [InlineData("double", "nint")]
+        [InlineData("double", "nuint")]
+        [InlineData("double", "float")]
+        [InlineData("double", "double")]
+        [InlineData("double", "decimal")]
+        [InlineData("decimal", "object")]
+        [InlineData("decimal", "string")]
+        [InlineData("decimal", "bool")]
+        [InlineData("decimal", "char")]
+        [InlineData("decimal", "sbyte")]
+        [InlineData("decimal", "short")]
+        [InlineData("decimal", "int")]
+        [InlineData("decimal", "long")]
+        [InlineData("decimal", "byte")]
+        [InlineData("decimal", "ushort")]
+        [InlineData("decimal", "uint")]
+        [InlineData("decimal", "ulong")]
+        [InlineData("decimal", "nint")]
+        [InlineData("decimal", "nuint")]
+        [InlineData("decimal", "float")]
+        [InlineData("decimal", "double")]
+        [InlineData("decimal", "decimal")]
```

Comment:
Consider using `CombinatorialData` (more compact, less tedious to type/review)

Reply:
> Consider using ```CombinatorialData``` (more compact, less tedious to type/review). This applies to other tests as well

Will simplify in one of the next PRs.

Reply:
Actually, I do not think ```CombinatorialData``` is going to work. We are not testing cross product of two type sets.